README.md

```diff
@@ -42,7 +42,18 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+
+var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
+
+
+
+    for (var i = 0; i <= foods.length; i++) {
+    console.log("I want to eat a " + foods[0][i]);
+
+
+    }
+
+
 ```
```

Comment:
:+1:  this works! can you think of another way to run this though? what would happen with your loop if we pushed "peaches" to the end of the array with fruits in it?